dds/DCPS/WriteDataContainer.cpp

```diff
@@ -642,6 +642,7 @@ WriteDataContainer::data_delivered(const DataSampleElement* sample)
                  OPENDDS_STRING(converter).c_str()));
     }
     release_buffer(stale);
+    stale = NULL;
```

Comment:
Shouldn't this be 0? NULL is normally not used in ACE/TAO and I assume also OpenDDS.